CSC8503/TutorialGame.cpp

```diff
@@ -397,7 +397,7 @@ GameObject* TutorialGame::AddSphereToWorld(const Vector3& position, float radius
 		.SetScale(sphereSize)
 		.SetPosition(position);
 
-	sphere->SetRenderObject(new RenderObject(&sphere->GetTransform(), sphereMesh, basicTex, nullptr, basicShader,radius));
+	sphere->SetRenderObject(new RenderObject(&sphere->GetTransform(), sphereMesh, mFloorAlbedo, mFloorNormal, basicShader,radius));
```

Comment:
Incase something prevents "RenderObject" to be created, compiler throws an error in the accurate line when we seperate object creations like this. In other words; it's easier to debug 🕺 

auto* renderObject = new RenderObject(&sphere->GetTransform();
sphere->SetRenderObject(renderObject , sphereMesh, mFloorAlbedo, mFloorNormal, basicShader,radius));